doc/developing-packages.rst

```diff
@@ -259,6 +259,54 @@ your libraries and executable sections. For example:
 Note that the ``import`` **must** be the first thing in the stanza. For more
 information see the :ref:`common-stanzas` section.
 
+Sets of packages that are verified to build together
+----------------------------------------------------
+
+Since Cabal 3.8, you can import from a
+:ref:`remote project configuration file<conditionals and imports>`:.
+This frees you from having to determine the acceptable version bounds for your
+dependencies, as described in the previous section.
+
+One provider of package sets is Stackage_. The Stackage snapshots contain a set
+of packages from Hackage that have all been verified to build with a given
+version of GHC.
+
+For example, the snapshot lts-19.2 contains versioned packages which all compile on GHC 9.0.
+
+If you use the following ``cabal.project`` file, the packages of the snapshot
+would be constrained to specific versions. The Cabal solver is then liberated
+from determining which versions to use, since it has already been determined
+which versions to use, at least for the packages in the snapshot.
+
+::
+
+    packages: .
+    import https://www.stackage.org/lts-19.2/cabal.config
+
+This is a tempting proposition, since you now only have one version number
+which specifies all of your dependencies' versions.
+
+A disadvantage of this approach is that consumers who don't use Stackage may
+suffer from the lacking bounds on your package. If published to Hackage, people
```

Comment:
The usage of lts does not imply necessarily not using bounds in .cabal files. In fact i would recommend use them anyways, in the form `^>=1.0 || ^>=2.1 || ...` if you want to support several ghcs/lts's. So you will have the lts and .cabal files checked one against the other and no stackage users will be happy

Reply:
This discussion as a whole is too gentle. Hackage uploads, by hackage policy, should conform to the pvp, full stop: https://hackage.haskell.org/upload

Reply:
Ok, I have added the suggestion by @gbaz in 79d84d59f31e31e425260e3290ae89b4ec8ab085.